service/passenger_service.py

```diff
@@ -0,0 +1,42 @@
+import textwrap
+
+from sqlalchemy import text, select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from model.drive_order import DriveOrder, DRIVE_ORDER_TABLE
+
+SAVE_RATING_QUERY_TEMPLATE = textwrap.dedent(
```

Comment:
unnecessary when you use sqlalchemy orm

Reply:
you are using it in rating service -whats the difference? should it be consistant through all services?